CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+## Creating releases
+
+If you are a maintainer of the repository and would like to tag a new release, follow these steps:
+
+- Send a heads-up in the Slack channels or post an issue with plan to tag a new release
+- Head over to [Actions](https://github.com/bazel-ios/rules_ios/actions)
+- Find the `Create release` action
+- Click on `Trigger workflow`
+- Type in a version number, follow SemVer whenever possible.
+- After a few minutes the release should show up in the [Releases](https://github.com/bazel-ios/rules_ios/releases) page.
```

Comment:
Thx for adding these! I think it's a good start to be able to manually release versions while we discuss what cadence to use.